app/src/androidTest/java/com/sdp/movemeet/viewTest/navigationTest/ActivityDescriptionNavigationTest.java

```diff
@@ -59,36 +63,43 @@ public void signIn(){
     @Test
     public void ActivityDescriptionToProfileActivity() {
```

Comment:
Why are those tests commented ?

Reply:
Because those are possibly the reason why we had the "Firebase Runloop" error...